hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AFileSystem.java

```diff
@@ -1473,12 +1473,12 @@ public long getDefaultBlockSize() {
   }
 
   private void printAmazonServiceException(AmazonServiceException ase) {
```

Comment:
What do you think about adding a warning or info message informing the user that an exception has been thrown and they can get more information by setting debug mode ?

Reply:
OK

Reply:
Fair enough !
Let's keep it this way